### ISSUE TITLE
Ng1ToNg2Module export missing

### ISSUE THREAD

**Issue by rmoscowitz** [ui-router/ng1-to-ng2#14]
My app is bundled with webpack and I've noticed that the imports are a little wonky.

Specifically, `Ng1ToNg2Module` seems to be missing from `ng1-to-ng2.min.js`. I have to go to the non-minified source to get it.

```
import { Ng1ToNg2Module } from 'ui-router-ng1-to-ng2/ng1-to-ng2.js';
```

When I log everything, it seems that Ng1ToNg2Module is missing:
```
import * as ng1ToNg2 from 'ui-router-ng1-to-ng2';
console.log(ng1ToNg2);
```
![image](https://cloud.githubusercontent.com/assets/3596827/18494182/40e96674-79da-11e6-9ffe-7db2cfa34616.png)

**Comment by rmoscowitz** [comment]
I'm on version 1.0.11.

**Comment by christopherthielen** [comment]
Wow, how does this even happen?

**Comment by christopherthielen** [comment]
Also worthy to note: After discussion with @adharris, we're going to be changing our NgModule approach for the next release.  The way it is currently implemented breaks AoT compilation.

**Comment by rmoscowitz** [comment]
"Wow, how does this even happen?" -- what I thought too! 😆 

With angular2 being released, I'm happy with waiting a bit and upgrading once everything's caught up. Thanks for responding though!

**Comment by christopherthielen** [comment]
@rmoscowitz can you try version 1.0.12 and let me know what you find

**Comment by rmoscowitz** [comment]
Is `"ui-router-ng1-to-ng2": "1.0.12",` compatible with `"@angular/core": "2.0.0-rc.5"`?

I'm getting this error, but I'm not 100% sure it's related to ng1-to-ng2.

![image](https://cloud.githubusercontent.com/assets/3596827/18845691/4c1a4d66-83e8-11e6-9633-ab89943e75b9.png)


I think I'm going to go ahead and skip rc5 and try with the release.

**Comment by christopherthielen** [comment]
version `1.0.12` requires angular `2.0.1` or higher

**Comment by rmoscowitz** [comment]
Ah ok, my bad. Npm did complain but I ignored it because I thought you meant to try 1.0.12 to see if this particular bug was fixed. I'll upgrade everything and give it a shot. Thanks!

**Comment by christopherthielen** [comment]
I'm interested to hear the outcome of the original bug report, if `Ng1ToNg2Module` exists or not in the minified file

**Comment by rmoscowitz** [comment]
Sorry for the delay! I can verify this is fixed in `"ui-router-ng1-to-ng2": "1.0.15",` with angular 2.0.2.
